app/client/components/cancel/cancellationReason.ts

```diff
@@ -9,16 +9,24 @@ export interface CancellationReason {
   alternateFeedbackIntro?: string;
   alternateFeedbackThankYouTitle?: string;
   alternateFeedbackThankYouBody?: string;
+  hideContactUs?: boolean;
   skipFeedback?: boolean;
 }
 
 export type CancellationReasonId =
   | "mma_financial_circumstances"
   | "mma_payment_issue"
+  | "mma_article"
   | "mma_editorial"
   | "mma_benefits"
+  | "mma_value_for_money"
   | "mma_support_another_way"
+  | "mma_direct_debit"
+  | "mma_one_off"
+  | "mma_wants_monthly_contribution"
+  | "mma_wants_annual_contribution"
   | "mma_health"
   | "mma_break_from_news"
   | "mma_values"
-  | "mma_other";
+  | "mma_other"
+  | "";
```

Comment:
Why do we include an empty string here?

Reply:
this is to allow the CancellationReasonContext (see cancellationContexts.tsx) to be parameterised by the CancellationReasonId type but still have a valid defaultValue. There are potentially other ways to achieve this but this seemed the neatest, if it is too nasty I can have a go at refactoring it.

Reply:
I'm not keen on it because it makes the type definition of CancellationReasonId a bit meaningless...

Can we make null the default value, and do something like this:
_React.Context<CancellationReasonId | null>_